core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -472,12 +512,12 @@ private[ui] class StagePage(parent: StagesTab) extends WebUIPage("stage") {
         }}
         {if (hasInput) {
           <td sorttable_customkey={inputSortable}>
-            {inputReadable}
+            {s"$inputReadable / $inputRecords"}
```

Comment:
I suggested below related to a separate concern, but what about just setting this to 0 instead of accumulating more and more negative values?